### PR TITLE
Create separate team page

### LINKED ISSUE #16: make separate page for team

### PR DESCRIPTION
**Pull request by eehitray**
Fixes #16 by adding a new team page to the website.

**Detailed changelog:**

1. Remove team section from index page, and add new link to team page under the About section on the index page.
2. Create new team page with short mockup intro text to team, along with team member profiles. Also adjusted flexbox item properties to space out the member profiles more evenly.

### CODE REVIEW

**Comment by ahsan8244** [comment]
Good stuff! Before we move on with this, you'll have to rebase your branch with master because some changes were made

File: src/pages/team.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+// Enable smooth-scroll behaviour
+if (typeof window !== "undefined") require("smooth-scroll")('a[href*="#"]');
+
+const TeamPage = ({ location }) => (
+  <>
+    <Layout location={location}>
+      <SEO title="Team" />
+      <Team />
+      <Contact />
```
Comment (welvin21) [inline src/pages/team.tsx]: We could remove the `Contact` component as it's already there in `index` page

Comment (eehitray) [reply src/pages/team.tsx]: Can do; just felt a contact section on each page would look better. @ahsan8244 thoughts?

**Comment by eehitray** [comment]
Removed the button in the about section and changed it to a navbar link in a07d032.

File: src/pages/team.tsx (hunk 1 above)
Comment (utkarsh867) [reply src/pages/team.tsx]: We can move that to the layout component if you all decide Contact Us to be on all pages.

File: src/components/team/member.tsx [hunk 2]
```diff
@@ -39,7 +39,14 @@ export const Member: FunctionComponent<MemberComponentProps> = ({
   const { name, picture } = memberData;
 
   return (
-    <Flex m={5} direction="column" align="center">
+    <Flex
```
Comment (utkarsh867) [inline src/components/team/member.tsx:42]: Why are we not using a Grid instead?

Comment (eehitray) [reply src/components/team/member.tsx:42]: Addressed in a5da481.

File: src/pages/team.tsx (hunk 1 above)
Comment (ahsan8244) [reply src/pages/team.tsx]: Let’s keep it at just the home page for now. We’ll add contact info to the footer as well

Comment (eehitray) [reply src/pages/team.tsx]: Addressed in 50eb8d5.

**Comment by utkarsh867** [comment]
How is this not merged yet?

@ahsan8244 @welvin21 @greatblitz982

**Comment by eehitray** [comment]
Not sure; all required changes have been made, waiting for a final review and close from the team.

**Review by welvin21 — APPROVED** [review]

**Pull request merged**